support/builder/config.sh

```diff
@@ -31,6 +31,9 @@ mkdir -p /hab/svc/builder-api
 cat <<EOT > /hab/svc/builder-api/user.toml
 log_level = "debug,tokio_core=error,tokio_reactor=error,zmq=error"
 
+[http]
+handler_count = 15
```

Comment:
do we actually need this?  the updated default is cpu * 2 which for dev will be smaller

Reply:
It was crashing on my dev machine with that many for some reason.

Reply:
argh ok, its actually default of cpu * 8 in the api.. we should fix that